Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,120 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    // Have set default walk and run speed so it looks about right with the animations
+    public float walkSpeed = 2.0f; 
+    public float runSpeed = 4.0f;
+    public float jumpHeight = 5.0f;
+    public float cameraHeight = 15.0f;
+    public float cameraDistance = 7.5f;
+    public bool invertCameraXAxis = false;
+    public bool invertCameraYAxis = false;
+    public float rotationSpeed = 120.0f;
+    public Camera mainCamera;
+    private Animator anim;
+    
+    private float moveSpeed;
+
```

Comment:
NIT: multiple line breaks, could we in a future issue remove these 🥺

Reply:
We're not going to care **_that_** much about code neatness, but can try and keep it tidier going forwards!